.swiftlint.yml

```diff
@@ -0,0 +1,38 @@
+opt_in_rules: # some rules are only opt-in
+  - empty_count
+included:
+  - IGListKitExamples
+  - IGListKitMessageExample
+  - IGListKitTodayExample
+excluded:
+  - Pods
+  - IGListKitExamples-UITests
```

Comment:
Habit --

My team prefers excluding test targets because we tend to bang everything in unit/ui tests. 

I can delete that line, however. 👌🏽